source/index.md

```diff
@@ -568,26 +566,76 @@ otherwise, exactly one payment method must be used.
 
 ##### Method: `applepay`
 
-<dl class="dl-horizontal">
-  <dt>applepay[token]</dt>
-  <dd>[:json:] <br /> Full, raw <code>PKPaymentToken</code> object, UTF-8 encoded serialization of a JSON dictionary.</dd>
-  <dt>applepay[symmetric_key]</dt>
-  <dd>[:hex:] <br /> EC symmetric AES key that can decrypt <code>data</code>
-      from the <code>PKPaymentToken</code>; hex formatted as it is a SHA256.</dd>
-</dl>
+Apple Pay requires data of the payment token to be decrypted. For this, the
+merchant's private key is needed to derive a symmetric key which is then used to
+decrypt the PAN, expiry etc.
+
+You can either
+
+* send us the private key and the merchant id (either directly or
+  indirectly by sending the merchant's certificate), then we will derive
+  the symmetric key and decrypt the payment details; or
+
+* derive the symmetric key from the payment token's ephemeral public key
+  together with the merchant's private key and merchant id/certificate.
```

Comment:
Suggestion:

```
You can either send us

* the private key and the merchant ID (either directly in <code>applepay[merchant_id]</code> or indirectly by sending us the merchant's certificate) or
* the symmetric key.

For the former option we derive the symmetric key and decrypt the payment details. For the latter option you need to derive the symmetric key, which is done using the payment token's ephemeral public key, the merchant's private key and the merchant ID.
```

Reply:
Will update `s/erchant id/erchant ID/` elsewhere too, and consider writing "Apple Pay merchant ID" where I think it's appropriate (since "merchant ID" is actually both MID in the context of 3DS and also wrt. Apple Pay, but it's two different IDs).